src/translate.directive.ts

```diff
@@ -27,6 +28,13 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
                 this.checkNodes(event.translations);
             });
         }
+
+        // subscribe to onDefaultLangChange event, in case the default language changes
+        if(!this.onDefaultLangChangeSub) {
+            this.onDefaultLangChangeSub = this.translateService.onDefaultLangChange.subscribe((event: DefaultLangChangeEvent) => {
+                this.checkNodes(event.translations);
```

Comment:
Yes, you can do that, I'll take a look at that problem when I have the time, because we probably need a better dom management to support nativescript and such